redis/release.lua

```diff
@@ -0,0 +1,15 @@
+local zset_key = KEYS[1]
+local worker_queue_key = KEYS[2]
+local owners_key = KEYS[3]
+
+-- owned_tests = {"SomeTest", "worker:1", "SomeOtherTest", "worker:2", ...}
+local owned_tests = redis.call('hgetall', owners_key)
+for index, owner_or_test in ipairs(owned_tests) do
+  if owner_or_test == worker_queue_key then -- If we owned a test
+    local test = owned_tests[index - 1]
+    redis.call('zadd', zset_key, "0", test) -- We expire the lease immediately
+    return nil
+  end
+end
```

Comment:
Nit: Is it worth iterating by step of 2 and only performing the equality check on the worker queue keys instead? Not sure if the performance implications are worth it, but would reduce # of elementary operations by half.

Reply:
I wanted to do that but couldn't figure out how to do it.

If you know, I can update.

Reply:
I'm not well-versed in Lua, so after digging into it I realized it is harder than I initially trivialized it to be.

That said, we might be able to query Redis for the length of the hash first, and use that to index into the response of `hgetall`.

Something like:

```lua
-- Can perform this in a transaction (i.e MULTI, EXEC) to guarantee atomicity so len of hash doesn't change
local len = redis.call('hlen', owners_key)
local owned_tests = redis.call('hgetall', owners_key)

for i=1,len do
  tuple_index = i * 2
  local owner = owned_tests[tuple_index + 1]
  if owner == worker_queue_key then
    local test = owned_tests[tuple_index]
    redis.call('zadd', zset_key, "0", test)
    return nil
  end
end
```

Not sure if the extra call to Redis is worth it. If you think this looks suitable, I can quickly throw a PR up